tests/e2e/registry/test_registry_image_push_pull.py

```diff
@@ -12,8 +12,7 @@ class TestRegistryImagePullPush(E2ETest):
     """
     Test to check Image push and pull worked with registry backed by OCS
     """
-
-    @workloads
+    # Remove this test from workloads due to https://github.com/red-hat-storage/ocs-ci/issues/1829
```

Comment:
@tiffanyn108 why not removing this test altogether?

Reply:
@tiffanyn108 Yes, please remove this test altogether.